frontend/dockerfile/docs/reference.md

```diff
@@ -842,7 +842,7 @@ environment variable with the same name.
 # syntax=docker/dockerfile:1
 FROM alpine
 RUN --mount=type=secret,id=API_KEY,env=API_KEY \
-    some-command --token-from-env API_KEY
+    some-command --token-from-env $API_KEY
```

Comment:
iiuc this is the name of the var that would then be solved within `some-command` logic. cc @dvdksn

Reply:
yeah, that was the idea, but I admit that it isn't really a great example. Any better suggestions are welcome

Reply:
thanks for reviewing, and I see.
I have no idea with alternative great example for now🙏